grpc/src/compiler/cpp_generator.cc

```diff
@@ -64,81 +65,7 @@ void PrintIncludes(grpc_generator::Printer *printer,
   }
 }
 
-grpc::string GetHeaderPrologue(grpc_generator::File *file,
```

Comment:
This seems like a large change, is this intended to be in this PR?

Reply:
Best I can suggest is to manually move the functions back where they were to see the actual in-source changes.
I think for this last commit, I just rearranged the functions.
I could back out this commit, add static to appropriate functions,
and do a separate PR where the functions are ONLY moved.

Reply:
That might be better.